### PR TITLE
[Enhancement] - removed most funded and fixed the tabs style

### PR DESCRIPTION
**Pull request by Rcureton**
## What
- Removed the most funded from sort options.
- Design sign off from @dannyalright 

## See 👀
most_funded

### CODE REVIEW

File: app/src/main/res/layout/discovery_layout.xml [hunk 1]
```diff
@@ -55,7 +55,7 @@
           android:layout_height="?android:attr/actionBarSize"
           android:layout_gravity="bottom"
           app:tabBackground="@drawable/click_indicator_light"
-          app:tabMode="scrollable"
+          app:tabMode="fixed"
```
Comment (eoji) [inline app/src/main/res/layout/discovery_layout.xml]: i'm cry

File: app/src/test/java/com/kickstarter/viewmodels/DiscoveryViewModelTest.java [hunk 2]
```diff
@@ -347,7 +347,7 @@ public void testClearingPages() {
         .build()
     );
 
-    this.clearPages.assertValues(Arrays.asList(0, 1, 2, 3), Arrays.asList(0, 2, 3, 4));
+    this.clearPages.assertValues(Arrays.asList(0, 1, 2, 3), Arrays.asList(0, 2, 3));
```
Comment (eoji) [inline app/src/test/java/com/kickstarter/viewmodels/DiscoveryViewModelTest.java]: o_O what is this testing? or rather, why does removing 4 make this test pass? O_o

Comment (Rcureton) [reply app/src/test/java/com/kickstarter/viewmodels/DiscoveryViewModelTest.java]: Have no idea at all I was actually confused about this test 🧐

**Comment by eoji** [comment]
[Remove Most Funded from sorts](https://trello.com/c/NX2J2YkZ/1015-remove-most-funded-from-sorts)

**Pull request merged**